frontend/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,256 @@
+import { ParamListBase, useNavigation } from "@react-navigation/native";
+import { StackNavigationProp } from "@react-navigation/stack";
+import { useEffect, useRef, useState } from "react";
+import {
+  View,
+  Text,
+  Image,
+  TouchableOpacity,
+  ScrollView,
+  StyleSheet,
+  Animated,
+  Dimensions,
+} from "react-native";
+import { Icon } from "react-native-elements";
+
+import theme from "../../constants/theme";
+import { navigationLinks } from "../../navigation/navigationLinks";
+import { useResponsiveLayout } from "../../utils/useResponsiveLayout";
+
+const MOBILE_HEADER_HEIGHT = 110;
```

Comment:
Could you match all the sizes to Figma?
MOBILE_HEADER_HEIGHT = 84
DESKTOP_HEADER_HEIGHT = 96
etc